.travis.yml

```diff
@@ -88,19 +94,13 @@ jobs:
     - name: "Release"
       stage: release
       name: "Push to Nexus"
-      if: commit_message ~= /\[trigger release\]/ AND branch ~= /^(master|SP\/.+|HF\/.+)$/
+      if: fork=false AND commit_message ~= /\[trigger release\]/ AND branch ~= /^(master|SP\/.+|HF\/.+)$/
```

Comment:
When we trigger a release we are only pushing to Nexus?

Reply:
yes this is the approach, acs use the share.war.

Reply:
So we will not create a docker image with this war? At least on quay?